srcpkgs/kirigami-addons/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'kirigami-addons'
+pkgname=kirigami-addons
+version=0.6.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF -DBUILD_QCH=ON"
+hostmakedepends="extra-cmake-modules gettext"
+makedepends="ki18n-devel kirigami2-devel"
+depends="kirigami2"
+short_desc="Add-ons for the Kirigami framework"
+maintainer="José Santos <agarimos@tutanota.com>"
+license="GPL-3.0-or-later"
+homepage="https://invent.kde.org/libraries/kirigami-addons"
+distfiles="${KDE_SITE}/stable/${pkgname}/${pkgname}-${version}.tar.xz"
```

Comment:
```suggestion
distfiles="${KDE_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
```

Your distfile deos expand to `https://download.kde.org/stable/stable/kirigami-addons/kirigami-addons-0.6.2.tar.xz` , so one `/stable` too much.

Do you have a custom set KDE_SITE by a chance, probably the reason it might work for you.